src/catalogs/synchronous/catalogs.test.ts

```diff
@@ -0,0 +1,92 @@
+import { Braze } from '../../Braze'
+import { request } from '../../common/request'
+import {
+  CatalogListItem,
+  CatalogListItemResponse,
+  CatalogListItemsResponse,
+  CatalogListResponse,
+} from './types'
+
+jest.mock('../../common/request/request')
+const mockedRequest = jest.mocked(request)
+
+const apiUrl = 'https://rest.iad-01.braze.com'
+const apiKey = 'apiKey'
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('Catalogs - Synchronous', () => {
+  const braze = new Braze(apiUrl, apiKey)
+  const options = {
+    headers: {
+      Authorization: `Bearer ${apiKey}`,
+      'Content-Type': 'application/json',
+    },
+  }
+
+  describe('List Catalogs', () => {
+    const response: CatalogListResponse = { catalogs: [], message: 'success' }
+    it('with no params', async () => {
```

Comment:
Nit: could we update the test description so it's more readable here and below?

```suggestion
    it('is called with no params', async () => {
```

Reply:
updated